src/test/java/elizCurtnazar/tests/BothPagesTests.java

```diff
@@ -0,0 +1,37 @@
+package elizCurtnazar.tests;
+
+import elizCurtnazar.driver.BrowserManager;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BothPagesTests {
+    public static void main(String[] args) {
+        ChromeDriver driver = BrowserManager.createDriver();
+//        TestPagesHerokuappManager herokuapp = new TestPagesHerokuappManager("https://testpages.herokuapp.com/styled/index.html", driver);
+        DemoQaManager demoqaPage = new DemoQaManager("https://demoqa.com/elements", driver);
+
+
+        demoqaPage.selectTextBox();
+        demoqaPage.selectCheckBox();
+        demoqaPage.selectRadioButton();
+        demoqaPage.WebTablesLink("webtables");
+        demoqaPage.ButtonsLink("buttons");
+        demoqaPage.LinkElementLink("links");
+
+
+//        herokuapp.goToTheFirstPage();
```

Comment:
Why are these lines commented? They shouldn't be in order to run the methods for heroku.

Reply:
done